solutions_builder/template_root/terraform/modules/vpc_network/main.tf

```diff
@@ -43,3 +43,16 @@ module "vpc" {
   }
 
 }
+
+module "cloud-nat" {
```

Comment:
is this needed?

Reply:
Cloud NAT is needed for the bastion host to connect to the internet